<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="cam_raw">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="cam_proc">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_controller_status">
      <property name="text">
       <string>Controller Status</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_temp_status">
      <property name="text">
       <string>TEMPERATURE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pressure_status">
      <property name="text">
       <string>PRESSURE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_acc_status">
      <property name="text">
       <string>ACCELEROMETER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_gravity_status">
      <property name="text">
       <string>GRAVITY</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_gyro_status">
      <property name="text">
       <string>GYROSCOPE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_mag_status">
      <property name="text">
       <string>MAGNETOMETER</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menurov_data">
    <property name="title">
     <string>rov data</string>
    </property>
   </widget>
   <addaction name="menurov_data"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
